hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/authorization/ValidateAccessToken.java

```diff
@@ -40,8 +45,14 @@ public ValidateAccessToken(AuthorizationProperties authorizationProperties, Stri
     @Override
     public void process(Exchange exchange)
             throws Exception {
-        AccessToken accessToken = AccessToken.parse(exchange.getIn().getHeader("Authorization").toString());
-        logger.info(String.format("Access token: %s", accessToken));
+    	String methodName = "process";
+    	String authorizationKey = (String) exchange.getIn().getHeader(AUTH_HEADER_KEY);
+    	if(authorizationKey==null) {
+    		logger.info("{} - No authorization key passed in request header.", methodName);
```

Comment:
Spacing/indentation looks off.

Reply:
I did Ctrl+i for whole file. Hoping it did not make it hard for review. This leads to another point, should we create a common formatter? How about: Create a task in this sprint to 
1. Create common formatter, 
2. Format all code in repo with that formatter and push in main

Reply:
I'll look into the Google Java Format plugin for Eclipse.